third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-emulate-run.js

```diff
@@ -16,7 +16,6 @@
   TestRunner.addResult('\n=============== Lighthouse Results ===============');
   TestRunner.addResult(`URL: ${lhr.finalUrl}`);
   TestRunner.addResult(`Version: ${lhr.lighthouseVersion}`);
-  TestRunner.addResult(`TestedAsMobileDevice: ${artifacts.TestedAsMobileDevice}`);
```

Comment:
print something else related to emulation here?

Reply:
done. [`af2946d` (#11779)](https://github.com/GoogleChrome/lighthouse/pull/11779/commits/af2946dffd81aa9ce69b106acfa7c4db7758742e)